docker-compose.yml

```diff
@@ -37,7 +39,9 @@ services:
       dockerfile: Dockerfile.test
       args:
         GOLANG_VERSION: 1.19
-    command: ["./scripts/test_e2e.sh"]
+    command: "/bin/sh -c 'cd e2etests && go test --tags e2etests'"
+    volumes:
+      - ${GO_MODCACHE}:/go/pkg/mod
```

Comment:
Won't this fail if GO_MODCACHE is not set?

Reply:
No, you get a warning that it is not set, and then it just downloads the mods.

Reply:
Hmm! I tried locally. I am getting the following error if GO_MODCACHE is not set or empty

```
WARNING: using --exit-code-from implies --abort-on-container-exit
Creating athense2e_teste2e_1 ... error

ERROR: for athense2e_teste2e_1  Cannot create container for service teste2e: create .: volume name is too short, names should be at least two alphanumeric characters

ERROR: for teste2e  Cannot create container for service teste2e: create .: volume name is too short, names should be at least two alphanumeric characters
ERROR: Encountered errors while bringing up the project.
make: *** [Makefile:76: test-e2e-docker] Error 1
➜  athens git:(fix-docker-e2e) ✗ 
```

Reply:
Fair